packages/esm-patient-programs-app/src/programs/program-actions-menu.component.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { showModal } from '@openmrs/esm-framework';
+import { ProgramsActionsMenu } from './programs-actions-menu.component';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  showModal: jest.fn(),
+  useLayoutType: jest.fn(() => 'desktop'),
+}));
```

Comment:
```suggestion
const mockShowModal = jest.mocked(showModal);
const mockUseLayoutType = jest.mocked(useLayoutType);
```

And then further below you could do this inside a `beforeEach` block:

```tsx
beforeEach(() => {
	mockUseLayoutType.mockReturnValue('small-desktop'); // or 'large-desktop' or 'tablet' 
})

To override the `useLayoutType` implementation to suit your test.